types/proper-lockfile/index.d.ts

```diff
@@ -0,0 +1,38 @@
+declare function lock (file: string, options?: Options): Promise<release>
```

Comment:
The smoking gun:

![image](https://user-images.githubusercontent.com/665810/105534950-218c2c00-5ce6-11eb-88d9-8ecb62c02d61.png)

Can these types not be added to `proper-lockfile`?